Formula/gromacs.rb

```diff
@@ -25,7 +25,10 @@ class Gromacs < Formula
   depends_on "gcc" # for OpenMP
```

Comment:
We prefer dependencies to be explicit than transitive, when there is a direct dependency.

Reply:
There is no choice to be had: formulas don't really have options, and binaries shipped are built on our CI machines.

Reply:
I think you are saying
* the gromacs formula declares a dependency on `openblas`, so it can and should encode decisions needed to build compatibly, per the openblas formula
* gromacs should therefore declare a dependency on `libgomp`
* the formula encodes its toolchain, which in this case must be gcc based

Are all three of those points correct?

I'm particularly unclear on that last one, since it would seem to invalidate the whole premise of this pull request (which has not yet been outright rejected). It seems like any one of those points, though, would invalidate much of this change.

Reply:
Yes (except there is no `libgomp`, it is a part of `gcc`). I have not outright closed the PR because I wanted to see if any other maintainer felt differently about it, but it seems not, so I'll close it now.